lib/StatusIndicator/styles.scss

```diff
@@ -2,30 +2,55 @@
    Config
    ========================================================================== */
 
-$status-indicator-size: 1em !default;
 $status-indicator-label-color--text: rgb(111, 121, 129) !default;
 
 /* ==========================================================================
    Styles
    ========================================================================== */
 
 .status-indicator {
-  display: inline-block;
-  vertical-align: middle;
   color: $status-indicator-label-color--text;
+  .due-date {
+    color: #8492a6;
```

Comment:
Always try to use colour variables from `settings/_colours.scss`.

This can be tricky at times but every single colour used should be derived from an abstracted variable defined in this settings file. Failing to do so will lead to multiple colours being added in a adhoc manner. This has already started to happen before so let's keep this in check.

If a colour variable doesn't exist then have a chat with the designer and come up with abstract names for them. Here are some examples;

- a yellow use for the workflow date: `$color-background-yellow` might be a good one to add.
- a new blue colour that is slightly off from the existing brand blue: use the existing `$color-background-blue` variable and speak with the team about this as we don't want to add multiple variations of a single colour without purpose.

**What happens when that new tint has a purpose?**

- create a new variable and add the purpose in the name e.g. a lighter tint of blue to help support interactions without being as powerful as the default brand blue, add a  `$color-background-blue-light` variable.